### PR TITLE
[xmpp] Add action for sending a group message

### PR DESCRIPTION
**Pull request by akallabeth**
my take on getting group messages to work.

did avoid updating smack (don´t know what that would trigger) and simply added a new function to be called

### CODE REVIEW

**Comment by akallabeth** [comment]
ok, did have a bug with not properly joining the group chat, but now it is working nicely on my local installation.

**Comment by akallabeth** [comment]
added optional setting of nickname for group chat as well now.

File: bundles/org.openhab.binding.xmppclient/src/main/java/org/openhab/binding/xmppclient/internal/client/XMPPClient.java [hunk 1]
```diff
@@ -73,14 +78,20 @@ public void unsubscribe(XMPPClientMessageSubscriber channel) {
...
-    public void connect(String host, Integer port, String login, String domain, String password,
+    public void connect(String host, Integer port, String login, String nick, String domain, String password,
             SecurityMode securityMode) throws XMPPClientConfigException, XMPPClientException {
         disconnect();
         String serverHost = domain;
         if (!host.isBlank()) {
             serverHost = host;
         }
 
+        if (!nick.isEmpty()) {
```
Comment (lsiepel) [inline bundles/org.openhab.binding.xmppclient/src/main/java/org/openhab/binding/xmppclient/internal/client/XMPPClient.java:89]: Guess you don't want whitespace names
```suggestion
        if (!nick.isBlank()) {
```

**Review by lsiepel — CHANGES_REQUESTED** [review]
Thanks for this contribution. Just one comment.

File: bundles/org.openhab.binding.xmppclient/src/main/java/org/openhab/binding/xmppclient/internal/client/XMPPClient.java (hunk 1 above)
Comment (akallabeth) [reply bundles/org.openhab.binding.xmppclient/src/main/java/org/openhab/binding/xmppclient/internal/client/XMPPClient.java]: good point, updated.

**Comment by akallabeth** [comment]
@lsiepel how can the ci be retriggered? looks like a glitch in the last run.

**Review by lsiepel — APPROVED** [review]
Thanks, LGTM

**Pull request merged**